Source/AtlusScriptLibrary/FlowScriptLanguage/Compiler/IntrinsicSupport.cs

```diff
@@ -52,31 +52,31 @@ public IntrinsicSupport(Library registry)
         PrintIntFunctionIndex = GetIndex(functions, "PUT");
         PrintStringFunctionIndex = GetIndex(functions, "PUTS");
         PrintFloatFunctionIndex = GetIndex(functions, "PUTF");
-        SupportsTrace = PrintIntFunctionIndex != -1 && PrintStringFunctionIndex != -1 && PrintFloatFunctionIndex != -1;
+        SupportsTrace = PrintIntFunctionIndex != 0xffff && PrintStringFunctionIndex != 0xffff && PrintFloatFunctionIndex != 0xffff;
```

Comment:
I would propose to change these to ``int`` instead to make the handling of -1 less awkward.

Reply:
`Instruction.COMM` expects a `ushort` for its param, meaning changing the type of function indexes in `IntrinsicSupport.cs` causes a compile error when they are passed to it in `FlowScriptCompiler.cs`. Since they are unsigned, they will never equal -1, so my thinking was to use `ushort.MaxValue` instead under the assumption that no flow function would actually have that as an id (I probably should have just typed `ushort.MaxValue` instead of `0xffff` though). I suppose I could alternatively explicitly convert when getting the support flag values, if that would be clearer.

Reply:
I was thinking of casting to ushort when they are used, but using ushort.MaxValue is fine to me too given that the check is only done once and stored as a bool anyway.